Author: Scott Salley <ssalley@beyondtrust.com>
Description: Correct function name and parameters that changed in krb5 1.9
Forwarded: not-needed
Last-Update: 2011-07-15

diff -Nurb likewise-open-6.1.0.249/lwio/server/smbcommon/smbkrb5.c likewise-open-6.1.0.249.new/lwio/server/smbcommon/smbkrb5.c
--- likewise-open-6.1.0.249/lwio/server/smbcommon/smbkrb5.c	2011-07-15 14:32:24.000000000 -0700
+++ likewise-open-6.1.0.249.new/lwio/server/smbcommon/smbkrb5.c	2011-07-15 15:08:49.000000000 -0700
@@ -333,9 +333,9 @@
                 authDataBuffer.value = &authData;
                 authDataBuffer.length = sizeof(authData);
                 
-                dwMajorStatus = gssspi_set_cred_option(
+                dwMajorStatus = gss_set_cred_option(
                     (OM_uint32 *)&dwMinorStatus,
-                    pContext->credHandle,
+                    &pContext->credHandle,
                     (gss_OID) &gssCredOptionPasswordOidDesc,
                     &authDataBuffer);
                 BAIL_ON_SEC_ERROR(dwMajorStatus);
